Pipfile.lock

```diff
@@ -231,10 +232,10 @@
         },
         "fakeredis": {
             "hashes": [
-                "sha256:8070b7fce16f828beaef2c757a4354af91698685d5232404f1aeeb233529c7a5",
-                "sha256:f8c8ea764d7b6fd801e7f5486e3edd32ca991d506186f1923a01fc072e33c271"
+                "sha256:01cb47d2286825a171fb49c0e445b1fa9307087e07cbb3d027ea10dbff108b6a",
```

Comment:
There weren't any changes to `Pipfile` so relocking is unnecessary. Please undo these changes to the lock file.